src/webui/src/components/trial-detail/TableList.tsx

```diff
@@ -599,7 +660,35 @@ class TableList extends React.Component<TableListProps, TableListState> {
                         selectionMode={SelectionMode.multiple}
                         selection={this.getSelectedRows}
                     />
+                {
+                      
+                    <Stack horizontal horizontalAlign="end" verticalAlign="baseline" styles={{root:{padding:10}}} tokens={horizontalGapStackTokens}>
+                        <Dropdown
+                        selectedKey={this.state.perPage === this.props.tableSource.length ? 'all':String(this.state.perPage)}
+                        options={perPageOptions}
+                        onChange={this.updateperPage}
+                        styles={{dropdown: { width: 150}}}/>
+
+                        {/* this.props.tableSource.length > this.state.perPage &&  */}
+                        <ReactPaginate
+                        previousLabel={"<"}
+                        nextLabel={">"}
+                        breakLabel={"..."}
+                        breakClassName={"break-me"}
+                        pageCount={this.state.pageCount}
+                        marginPagesDisplayed={2}
+                        pageRangeDisplayed={2}
+                        onPageChange={this.handlePageClick}
+                        containerClassName={(this.props.tableSource.length == 0 ? "pagination hidden" : "pagination" )}
+                        subContainerClassName={"pages pagination"}
+                        disableInitialCallback={false}
+                        activeClassName={"active"}/>
```

Comment:
string type could directly write like this breakClassName="break-me"? Such as other className, label type. pls check